fhirzeug/generators/python_pydantic/static_files/tests/test_examples/test_import_examples.py

```diff
@@ -1,5 +1,6 @@
 import re
 import json
+import _io
```

Comment:
Couldn't we import `io` instead and use `io.TextIOWrapper` below?

```suggestion
import io
```